charts/vald/values.yaml

```diff
@@ -1080,7 +1080,7 @@ agent:
   revisionHistoryLimit: 2
   # @schema {"name": "agent.terminationGracePeriodSeconds", "type": "integer", "minimum": 0}
   # agent.terminationGracePeriodSeconds -- duration in seconds pod needs to terminate gracefully
-  terminationGracePeriodSeconds: 120
+  terminationGracePeriodSeconds: 600
```

Comment:
I think, if we set terminationGracePeriodSeconds to 10minutes we should disable in-memory mode by default

Reply:
or enable in-memory mode & make terminationGracePeriod shorter. which one is better?
i think it's better that in-memory mode is enabled. so i'll revert the change for this line, ok?

Reply:
sounds good, we have to work on PV mode & S3 mode & combine Persistent mode document XD